{
  "id": "CVE-2021-20272",
  "tags": [
    {
      "label": "Product",
      "message": "privoxy"
    },
    {
      "label": "Version",
      "message": "n/a"
    },
    {
      "label": "Vulnerability",
      "message": "CWE-617"
    }
  ],
  "description": " A flaw was found in privoxy before 3.0.32. An assertion failure could be triggered with a crafted CGI request leading to server crash.",
  "poc": {
    "githubs": [
      "https://github.com/MegaManSec/privoxy"
    ]
  }
}
